R/stats_wrapper.R

```diff
@@ -1,6 +1,13 @@
 #'
 #'
 
+#' scale wrapper that returns a vector as a result
+#' @export
+scale_vec <- function(...) {
+  ret <- scale(...)
+  as.numeric(ret)
```

Comment:
Not very sure what is going on here.
Looks like only difference with regular scale() is as.numeric().
Does this mean as.numeric have side effect of making non-vector a vector?
Please explain a little more in the comment.